udunits2/udunits2.spec

```diff
@@ -0,0 +1,243 @@
+Name: udunits2
+Version: 2.2.28
+Release: 7%{?dist}
+Summary: A library for manipulating units of physical quantities
+License: UCAR
+URL: http://www.unidata.ucar.edu/software/udunits/
+Source0: ftp://ftp.unidata.ucar.edu/pub/udunits/udunits-%{version}.tar.gz
+# default_udunits2_xml_path() is buggy and broken
+Patch0: udunits-2.2.28-fix-xml-path-logic.patch
+BuildRequires: make
+BuildRequires: gcc-c++, groff, byacc, expat-devel, CUnit-devel
+BuildRequires: chrpath
+BuildRequires: /usr/bin/makeinfo
+BuildRequires: /usr/bin/texi2dvi
+# workaround touching configure during build by the %%configure macro on ppc64le RHEL 7
+# can go away when upstream refreshes the autoconf/libtool files
+%if 0%{?rhel} == 7
```

Comment:
I understand this .spec file is copied from somewhere, otherwise we wouldn't care about RHEL 7.
This `%if` does not change any behavior for openEuler, so it is OK to preserve this code.

Reply:
I am a beginner to write specs, most of which are reference online, thanks to martin-g's helpful guidance, I will learn about spec as soon as possible.